### DISCUSSION TITLE
MKV embedded subtitles

### DISCUSSION THREAD

**Discussion by m5k4** [smplayer-dev/smplayer#659] [Q&A]
Probably asked before, but can SMPlayer use the subtitles typically embedded in MKV files? From other video players I can select one of the available languages in the list. In SMPlayer I cannot.

**Comment by cipricus** [comment]
Yes, just like separate subtitles: you can select them under Subtitles option:

![image](https://user-images.githubusercontent.com/3447868/227612572-8a8cb335-1d6d-48b0-823d-128e60d1b74b.png)

You can also add a button to list them

![image](https://user-images.githubusercontent.com/3447868/227612789-237c9320-f434-47a6-9ee8-45fd1ceb1c0e.png)

or cycle them with a shortcut

![image](https://user-images.githubusercontent.com/3447868/227613217-7e130917-9556-409b-bc49-f2596a522543.png)

**Comment by m5k4** [comment]
Thanks man!

cheers
 Morten

fre. 24. mar. 2023 kl. 19:46 skrev cipricus ***@***.***>:
